### PR TITLE
Update FeatureTest.java

### PR DESCRIPTION
**Pull request by redox0018**

### CODE REVIEW

File: FeatureTest.java [hunk 1]
```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
         System.out.println(str.length());
-        str = "fj";
+        str = "fjff";
         return variable * variable;
     }
 }
```
Comment (redox0018) [inline FeatureTest.java:12]: Null pointer dereference: Variable 'str' is null (assigned null at line 7)

File: FeatureTest.java [hunk 2]
```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
```
Comment (redox0018) [inline FeatureTest.java:7]: Method parameter 'args' might be null - consider adding null check

**Review by redox0018 — COMMENTED** [review]
Automated code review completed with 2 new comments.
Please address the issues found.

File: FeatureTest.java (hunk 1 above)
Comment (redox0018) [inline FeatureTest.java:12]: [code-reviewer[bot]]
Null pointer dereference: Variable 'str' is null (assigned null at line 7)

File: FeatureTest.java (hunk 2 above)
Comment (redox0018) [inline FeatureTest.java:7]: [code-reviewer[bot]]
Method parameter 'args' might be null - consider adding null check

**Review by redox0018 — COMMENTED** [review]
[code-reviewer[bot]]
Automated code review completed with 2 new comments.
Please address the issues found.

**Review by piramal-codeReviewer — COMMENTED** [review]
[code-reviewer[bot]]
Automated code review completed with 2 new comments.
Please address the issues found.

File: FeatureTest.java (hunk 1 above)
Comment (piramal-codeReviewer) [inline FeatureTest.java:12]: [code-reviewer[bot]]
Null pointer dereference: Variable 'str' is null (assigned null at line 7)

File: FeatureTest.java (hunk 2 above)
Comment (piramal-codeReviewer) [inline FeatureTest.java:7]: [code-reviewer[bot]]
Method parameter 'args' might be null - consider adding null check

**Review by piramal-codeReviewer — COMMENTED** [review]
[code-reviewer[bot]]
Automated code review completed with 2 new comments.
Please address the issues found.

File: FeatureTest.java (hunk 2 above)
Comment (redox0018) [reply FeatureTest.java:7]: resolved

[the remark this answers is not in the document]
Comment (redox0018) [reply FeatureTest.java:7]: resolved

File: FeatureTest.java [hunk 3]
```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
...
         System.out.println(str.length());
-        str = "fj";
+        str = "fjff";
```
Comment (redox0018) [inline FeatureTest.java:9]: fddvdgdgdg

Comment (redox0018) [inline FeatureTest.java:9]: dfddgdgfdgfdgdf

File: FeatureTest.java (hunk 1 above)
Comment (piramal-codeReviewer) [inline FeatureTest.java:12]: [code-reviewer[bot]]
Null pointer dereference: Variable 'str' is null (assigned null at line 7)

File: FeatureTest.java (hunk 2 above)
Comment (piramal-codeReviewer) [inline FeatureTest.java:7]: [code-reviewer[bot]]
Method parameter 'args' might be null - consider adding null check

**Review by piramal-codeReviewer — COMMENTED** [review]
[code-reviewer[bot]]
Automated code review completed with 2 new comments.
Please address the issues found.

File: FeatureTest.java (hunk 1 above)
Comment (piramal-codeReviewer) [inline FeatureTest.java:12]: Null pointer dereference: Variable 'str' is null (assigned null at line 7)

File: FeatureTest.java (hunk 2 above)
Comment (piramal-codeReviewer) [inline FeatureTest.java:7]: Method parameter 'args' might be null - consider adding null check

**Review by piramal-codeReviewer — COMMENTED** [review]
[code-reviewer[bot]]
Automated code review completed with 2 new comments.
Please address the issues found.

**Pull request merged**